sentry-opentelemetry/sentry-opentelemetry-bootstrap/src/main/java/io/sentry/opentelemetry/OtelSpanFactory.java

```diff
@@ -137,8 +137,14 @@ public final class OtelSpanFactory implements ISpanFactory {
       if (description != null) {
         sentrySpan.setDescription(description);
       }
-      if (samplingDecision != null) {
-        sentrySpan.getSpanContext().setSamplingDecision(samplingDecision);
+      // TODO [POTEL] do we need this?
```

Comment:
setting sampling decision should no longer be necessary since we already attach it as separate attributes above